contracts/Removal.sol

```diff
@@ -270,6 +271,11 @@ contract Removal is
   ) public override {
     // todo perhaps call this listRemovals instead
     // todo do we add any validation to enforce that all removals in batch belong to the same project id?
+    for (uint256 i = 0; i < amounts.length; i++) {
```

Comment:
Kinda wonder if the better place for this is inside `_beforeTokenTransfer` where we check if amount is 0 as that would also prevent other cases where transfers of zero could occur